source/Jobbr.Server/Jobbr.Server.csproj

```diff
@@ -19,9 +19,6 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>pdbonly</DebugType>
   </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="Microsoft.CSharp" />
-  </ItemGroup>
```

Comment:
Are you sure about this? I keep forgetting when this is actually needed 🤔

Reply:
It's needed when using the dynamic keywork, which we do but it gives us an warning "No way to resolve conflict between "Microsoft.CSharp, Version=6.0.0.0 .." so it seems it's picked up from somewhere else